assets/js/uploader.js

```diff
@@ -0,0 +1,26 @@
+const s3Uploader = (entries, onViewError) => {
+
+  entries.forEach((entry) => {
+    let formData = new FormData()
+    let {url, fields} = entry.meta
+    Object.entries(fields).forEach(([key, val]) => formData.append(key, val))
+    formData.append("file", entry.file)
+    let xhr = new XMLHttpRequest()
```

Comment:
im sure this came from some example somewhere but i feel like we should just use `const` for everything here... not seeing the benefit of `let`